translation.yml

```diff
@@ -0,0 +1,7 @@
+source_language: en
+target_languages: [de, es, fr, it, ja]
+components:
+  - name: 'partner'
```

Comment:
I feel like the _users_ are merchants, since all of the strings are merchant-facing.

Reply:
Restored the initial state of this file 👍